README.md

```diff
@@ -5,39 +5,78 @@
 
-The purpose of this secondary logfile is twofold:
+This tool, henceforth referred to simply as "the runtime", builds upon
+the [virtcontainers](https://github.com/containers/virtcontainers)
+project to provide a high-performance standards-compliant runtime that
+creates hardware-virtualized containers which leverage
+[Intel](https://www.intel.com/)'s VT-x technology.
 
-- To allow all log output to be recorded in a non-container-specific
-  path.
+It is a re-implementation of [`cc-oci-runtime`](https://github.com/01org/cc-oci-runtime) written in the go language and supersedes `cc-oci-runtime` starting from 3.0.0.
 
-  This is particularly useful under container managers such as Docker
-  where if a container fails to start, the container-specific directory
-  (which contains the `--log=` logfile) will be deleted on error, making
-  debugging a challenge.
+The runtime is both [OCI](https://github.com/opencontainers/runtime-spec)-compatible and [CRI-O](https://github.com/kubernetes-incubator/cri-o)-compatible, allowing it to work seamlessly with both Docker and Kubernetes respectively.
 
-- To collate the log output from all runtimes in a single place.
+## License
 
-The global logfile comprises one line per entry. Each line contains the
-following fields separated by colons:
+The code is licensed under an Apache 2.0 license.
 
-- timestamp
-- PID
-- program name
-- log level
-- log message
+See [the license file](https://github.com/clearcontainers/runtime/blob/master/LICENSE) for further details.
 
-The global logfile records all log output sent to the standard logfile.
+## Hardware requirements
 
-Note that if output is disabled for the standard logfile, the global log
-will still record all logging calls the runtime makes.
+The runtime has a built-in command to determine if your host system is capable of running a Clear Container. Simply run:
 
-The global logfile is disabled by default. It can be enabled either in
-the `configuration.toml` configuration file or by setting the
+```bash
+$ cc-runtime cc-check
+```
+
+## Quick start for developers
+
+See the [developer's installation guide](https://github.com/clearcontainers/runtime/blob/master/docs/developers-clear-containers-install.md).
+
+## Community
+
+See [the contributing document](https://github.com/clearcontainers/runtime/blob/master/CONTRIBUTING.md).
+
+## Configuration
+
+The runtime uses a single configuration file called `configuration.toml` which is normally located at `/etc/clear-containers/configuration.toml`.
+
+To see details of your systems runtime environment (including the location of the configuration file), run:
+
+```bash
+$ cc-runtime cc-env
+```
+
+## Debugging
+
+To provide a persistent log of all container activity on the system, the runtime
+provides a global logging facility. By default, this feature is disabled.
+
+To enable the global log:
+
+```bash
+$ sudo sed -i -e 's/^#\(\[runtime\]\|global_log_path =\)/\1/g' /etc/clear-containers/containers.toml
```

Comment:
That is handled by the note below, but yes, logically it should be handled first. Fixing...